### PR TITLE
Spectre meltdown intro

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+At the turn of the year 2017 and 2018 the world of security and computing has
+shaken. It was the time when we first heard about vulnerabilities that affect
+almost every modern processor (mainly x86 architecture) manufactured during the
+last 20 years. They have been named as Spectre and Meltdown and belong to one
+family of flaws caused by speculative execution. In this post, I will describe
+what they are and how they are threatening the users of modern machines.
+
+# Spectre
+
+![Spectre](https://3mdeb.com/wp-content/uploads/2017/07/spectre.png)
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: @pietrushnic license is CC0

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+malicious code resided in attackers address space. While in the second variant,
+the attacker chooses a specific "gadget" from victim's address space. The
+processor is trained to mispredict a branch and to execute the "gadget" lying
+in the victim's address space. Information revealed by the "gadget" remains in
+the cache like before and the attacker can analyze it to retrieve the
+information.
+
+## Variant 3 - Meltdown, rogue data cache load
+
+![Meltdown](https://3mdeb.com/wp-content/uploads/2017/07/meltdown.png)
```
Comment (miczyg1) [inline _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: Same here, CC0

Comment (pietrushnic) [reply _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: @miczyg1 that means we definitely can use that.

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+In the next post, I will show You whether these vulnerabilities are exploitable
+on PC Engines apu2 platform and how to mitigate. So stay tuned.
+
+I hope this post was useful for you. Please feel free to share your opinion and
+if you think there is value, then share with friends.
+
+If you think we can help in improving the security of your firmware or you
+looking for someone who can boot your product by leveraging advanced features
+of used hardware platform, feel free to drop us email to `contact<at>3mdeb.com`
```
Comment (pietrushnic) [inline _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: @miczyg1 please improve that as we discussed in CPB `feel free to [boot a call with us](https://calendly.com/3mdeb/consulting-remote-meeting) or drop us email to ... If you are interested in similar content feel free to [sing up to our newsletter](http://eepurl.com/gfoekD)`

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
+---
+post_title: Meltdown spectre vulnerabilities
```
Comment (pietrushnic) [inline _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: @miczyg1 let's improve this title. "What you should know about Spectre and Meltdown before patching?" - 20%

**Comment by pietrushnic** [comment]
@miczyg1 next steps:
1. Fix issues
2. Apply Yoas guidelines and fix most problems
3. Let's merge and notify @Moviss about blog post scheduling

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md (hunk 3 above)
Comment (miczyg1) [reply _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: Done

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md (hunk 4 above)
Comment (miczyg1) [reply _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: Volume: 22,200/mo | CPC: $4.09 | Competition: 0.04
for `meltdown and spectre`. As these are the first words appearing in the title, it should be good to go now. Changed title to `Meltdown and Spectre. What are they and what are they not?`

**Comment by miczyg1** [comment]
@pietrushnic fixed 1. and 2. Yoast reports OK. Draft available on WP for preview

File: _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md (hunk 4 above)
Comment (pietrushnic) [reply _posts/2019-01-XX-meltdown-and-spectre-vulnerabilities.md]: @miczyg1 super.

**Comment by miczyg1** [comment]
@pietrushnic if everything looks ok lets schedule publication.

**Comment by pietrushnic** [comment]
@miczyg1 please do - I'm not sure if I should merge that.

**Comment by miczyg1** [comment]
@pietrushnic I have scheduled the post for 20th of March, one week after latest post. If we were not supposed to merged that, I will fix it.

**Comment by pietrushnic** [comment]
@miczyg1 leave it as it is. We will move to Hugo soon, so whole publishing procedure will have to change.

**Pull request merged**